### PR TITLE
[5.5] DelegatesToResource: fix offsetExists.

### PR DESCRIPTION
**Pull request by lucasmichot**
DelegatesToResource: fix offsetExists.

### CODE REVIEW

File: src/Illuminate/Http/Resources/DelegatesToResource.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function resolveRouteBinding($value)
      */
     public function offsetExists($offset)
     {
-        return array_key_exists($this->resource[$offset]);
```
Comment (themsaid) [inline src/Illuminate/Http/Resources/DelegatesToResource.php]: I think it should have been 

```
return isset($this->resource[$offset])
```

File: src/Illuminate/Http/Resources/DelegatesToResource.php [hunk 2]
```diff
@@ -46,7 +46,7 @@ public function resolveRouteBinding($value)
...
+        return array_key_exists($this->resource, $offset);
```
Comment (themsaid) [inline src/Illuminate/Http/Resources/DelegatesToResource.php]: Also I believe you mean:

```
return array_key_exists($offset, $this->resource);
```

Comment (lucasmichot) [reply src/Illuminate/Http/Resources/DelegatesToResource.php]: Good catch, fixed: https://github.com/laravel/framework/pull/20887/commits/c907481d94bc9d9c80f3c5af1f6d63add23d1a31

Comment (laurencei) [reply src/Illuminate/Http/Resources/DelegatesToResource.php]: In all seriousness - why did the tests not catch that error?

**Comment by lucasmichot** [comment]
https://github.com/laravel/framework/pull/20887#discussion_r136458222

yeah @laurencei - some lack happens sometimes 😎 - and we can find some code errors sometimes
I believe a proper code coverage (codecov ?) should be set-up

**Pull request merged**